mealie/schema/group/group_shopping_list.py

```diff
@@ -225,10 +225,14 @@ class ShoppingListUpdate(ShoppingListSave):
 
 
 class ShoppingListOut(ShoppingListUpdate):
-    recipe_references: list[ShoppingListRecipeRefOut]
-    label_settings: list[ShoppingListMultiPurposeLabelOut]
+    recipe_references: list[ShoppingListRecipeRefOut] = []
+    label_settings: list[ShoppingListMultiPurposeLabelOut] = []
```

Comment:
I don't think you can use the array like this? Won't it always be a reference to the same array?

Reply:
Normally that's true, but Pydantic does some special magic and makes it safe: https://stackoverflow.com/a/63808835

Reply:
Thought this was in the models folder, ignore me! :D